### PR TITLE
upgrade ink; upgrade openbrush

### PR DESCRIPTION
**Pull request by boyswan**
Upgrade ink to `4.1.0`
Remove `3.0.0` openbrush tag and use latest (for 4.1.0 compatibility)

### CODE REVIEW

File: crates/catalog/Cargo.toml [hunk 1]
```diff
@@ -5,10 +5,10 @@ authors = ["Stake Technologies <devops@stake.co.jp>"]
 edition = "2021"
 
 [dependencies]
-ink = { version = "4.0.0", default-features = false }
+ink = { version = "4.1.0", default-features = false }
 scale = { package = "parity-scale-codec", version = "3", default-features = false, features = ["derive"] }
 scale-info = { version = "2.3", default-features = false, features = ["derive"], optional = true }
-openbrush = { tag = "3.0.0", git = "https://github.com/727-Ventures/openbrush-contracts", default-features = false, features = ["access_control", "reentrancy_guard", "psp34"] }
+openbrush = { git = "https://github.com/727-Ventures/openbrush-contracts", default-features = false, features = ["access_control", "reentrancy_guard", "psp34"] }
```
Comment (Maar-io) [inline crates/catalog/Cargo.toml:11]: it might be tricky to leave it on master. It might  bring compatibility issues very soon

**Review by Maar-io — APPROVED** [review]
LGTM

**Comment by Maar-io** [comment]
please rebase before merge.

File: crates/catalog/Cargo.toml (hunk 1 above)
Comment (boyswan) [reply crates/catalog/Cargo.toml]: Looks like 3.1.0 was released, set to the tag!

**Pull request merged**